### PR TITLE
HW01 Pavel Karenda

### PR DESCRIPTION
**Pull request by ButterBrot777**
In first task I implemented a class AngleCalculator. I am not sure I was correct in syntax and understand classes in Ruby correct. Give me please full feedback about this task solution.

### CODE REVIEW

**Comment by vladhilko** [comment]
Please, run rubocop

File: HW01/Pavel_Karenda/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.txt
+.DS_Store
```
Comment (vladhilko) [inline HW01/Pavel_Karenda/.gitignore]: We shouldn't push any additional files

File: HW01/Pavel_Karenda/pavel_karenda_hw_01_t01.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    @hours = hours
+    @minutes = minutes
+  end
+
+  def calc_angle
+    result = ((60 * hours + minutes) / 2 - 6 * minutes).abs
+    result = 360 - result if result > 180
+    result
+  end
+end
```
Comment (vladhilko) [inline HW01/Pavel_Karenda/pavel_karenda_hw_01_t01.rb]: It's a correct way of using class, don't worry 👍

File: HW01/Pavel_Karenda/.gitignore (hunk 1 above)
Comment (ButterBrot777) [reply HW01/Pavel_Karenda/.gitignore]: but... it's just .gitignore... could I not delete this file please?

Comment (vladhilko) [reply HW01/Pavel_Karenda/.gitignore]: Can you remove this `.gitignore` ?

Comment (vladhilko) [reply HW01/Pavel_Karenda/.gitignore]: ok

**Pull request merged**